samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp

```diff
@@ -774,9 +774,13 @@
                     <h:outputLabel for="feedbackCheckbox1" value="#{assessmentSettingsMessages.student_response}" />
                 </h:panelGroup>
                 <h:panelGroup styleClass="form-inline" layout="block">
-                    <h:selectBooleanCheckbox value="#{publishedSettings.showCorrectResponse}" id="feedbackCheckbox13"/>
+                    <h:selectBooleanCheckbox value="#{publishedSettings.showCorrectResponse}" id="feedbackCheckbox13" onclick="changeStatusCorrectResponseCheckbox()"/>
                     <h:outputLabel for="feedbackCheckbox3" value="#{assessmentSettingsMessages.correct_response}" />
                 </h:panelGroup>
+                <h:panelGroup styleClass="form-inline indent1" id="hideCorrectResponse" layout="block" style="display: #{(publishedSettings.showCorrectResponse) ? 'block' : 'none'}">
+                    <h:selectBooleanCheckbox value="#{publishedSettings.showCorrection}" id="feedbackCheckbox20"/>
+                    <h:outputLabel for="feedbackCheckbox9" value="#{assessmentSettingsMessages.show_correct_response}" />
```

Comment:
The feedbackCheckbox20 id is because the last checkbox was the feedbackCheckbox19 and to continue the logic of that configuration i put it like feedbackCheckbox20. And the feedbackCheckbox9 for it is related to the authorSettings.jsp, where that configuration was feedbackCheckbox9

Reply:
This has to be feedbackCheckbox20 right?

Reply:
I don't think this makes sense, I believe all the labels are wrong, the outputLabel needs to reference the input,

Label for feedbackCheckbox20, input with id feedbackCheckbox20 .

Reply:
No, It should been feedbackCheckbox19, sorry about the mistake